src/main/scala/tech/cryptonomic/conseil/tezos/TezosDatabaseOperations.scala

```diff
@@ -79,7 +79,7 @@ object TezosDatabaseOperations extends LazyLogging {
     * @return     Database action possibly containing the number of rows written (if available from the underlying driver)
     */
   def writeAccountsHistory(
-      accountsInfo: List[(BlockTagged[Map[AccountId, Account]], List[AccountsHistoryRow])]
+      accountsInfo: List[(BlockTagged[Map[AccountId, Account]], List[Tables.AccountsRow])]
```

Comment:
We might want to document this additional piece of input, to keep track of why we need this additional list of existing rows here...

Reply:
updated the docs